Last-Update: 2018-06-18
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: prevents test-time installation that causes FTBFS.
~~~~
 go test net: open /usr/lib/go-1.10/pkg/linux_amd64/net.a: permission denied
~~~~

--- a/hack/test/unit
+++ b/hack/test/unit
@@ -18,12 +18,8 @@
 
 exclude_paths="/vendor/|/integration"
 pkg_list=$(go list $TESTDIRS | grep -vE "($exclude_paths)")
 
-# install test dependencies once before running tests for each package. This
-# significantly reduces the runtime.
-go test -i "${BUILDFLAGS[@]}" $pkg_list
-
 for pkg in $pkg_list; do
     go test "${BUILDFLAGS[@]}" \
         -cover \
         -coverprofile=profile.out \
